### PR TITLE
house keeping

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**

### CODE REVIEW

File: contracts/core/Merge.sol [hunk 1]
```diff
@@ -13,17 +13,22 @@ contract Merge is IMerge, IWeweReceiver, IERC1363Spender, Ownable, ReentrancyGua
     using SafeERC20 for IERC20;
 
     IERC20 public immutable wewe;
-    IERC20 public immutable vult;
+    IERC20 public immutable vault;
```
Comment (hippocampus-web3) [inline contracts/core/Merge.sol:16]: this change does not apply, it is not a vault, but refers to the vultisig vult token https://vultisig.com/.

Comment (bitcoinbrisbane) [reply contracts/core/Merge.sol:16]: Ok, ill spell it out so its clear.

**Comment by hippocampus-web3** [comment]
@bitcoinbrisbane There are still quite a few references that have been renamed to Vault, but they have to be Vult because they refer to the Vultisig token

**Comment by bitcoinbrisbane** [comment]
> @bitcoinbrisbane There are still quite a few references that have been renamed to Vault, but they have to be Vult because they refer to the Vultisig token

Ah ok.  So we're not using the term vault at all?  I might drop this PR and lets discuss.

**Comment by bitcoinbrisbane** [comment]
-

**Pull request closed without merging**